src/config/env/prd.json

```diff
@@ -36,5 +36,6 @@
   "GATSBY_VOTING_POWER_TO_PASS_GOVERNANCE": "6000000",
   "GATSBY_VOTING_POWER_TO_PASS_POI": "500000",
   "GATSBY_VOTING_POWER_TO_PASS_POLL": "500000",
-  "GATSBY_GRANT_SIZE_MINIMUM": "1"
+  "GATSBY_GRANT_SIZE_MINIMUM": "1",
+  "DISCORD_SERVICE_ENABLED": "true"
```

Comment:
This variable is not necessary here as the discord service is only used server side 🤔